types/wlr_output.c

```diff
@@ -364,6 +364,7 @@ void wlr_output_init(struct wlr_output *output, struct wlr_backend *backend,
 		output->software_cursor_locks = 1;
 	}
 
+	wl_list_init(&output->display_destroy.link);
```

Comment:
This seems like an unrelated change. It also seems like a no-op since `wl_display_add_destroy_listener` overwrites it right after.

Reply:
Dropped it during commit rebase/split